test/unit-test/test-Model.js

```diff
@@ -0,0 +1,851 @@
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Uint8Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is FLOAT32 (0), setting its value as a Uint8ClampedArray is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Uint8ClampedArray(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is FLOAT32 (0), setting its value as a Int16Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Int16Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is FLOAT32 (0), setting its value as a Uint16Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Uint16Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is FLOAT32 (0), setting its value as a Int32Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Int32Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is FLOAT32 (0), setting its value as a Uint32Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Uint32Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
+        });
+      });
+    });
+
+    it('target operand type is FLOAT32 (0), setting its value as a Float32Array is ok', function() {
+      return nn.createModel().then((model)=>{
+        model.addOperand({type: nn.FLOAT32, dimensions: [1, 2, 2, 1], scale: 0.8, zeroPoint: 1});
+        let inputData = new Float32Array(4);
+        inputData.fill(0);
+        assert.doesNotThrow(() => {
+          model.setOperandValue(0, inputData);
```

Comment:
It should throw as it only accepts scalar Float32 value, so Float32Array length should be 1.